icu4c/source/i18n/unicode/reldatefmt.h

```diff
@@ -169,10 +169,9 @@ typedef enum UDateAbsoluteUnit {
      */
     UDAT_ABSOLUTE_NOW,
 
-#ifndef U_HIDE_DRAFT_API
```

Comment:
You should move this down, not delete it

Reply:
done